Solutions/Vellum.Cli/Vellum/Cli/Commands/Environment/SetEnvironmentSettingHandler.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// <copyright file="SetEnvironmentSettingHandler.cs" company="Endjin Limited">
 // Copyright (c) Endjin Limited. All rights reserved.
 // </copyright>
-
+/*
```

Comment:
Do we need all these commented out files?

Reply:
I'm just double checking I've migrated all the help text across, once done, I will remove them